REFERENCE.md

```diff
@@ -0,0 +1,195 @@
+# Learnosity Node.js-SDK: Reference guide
+
+## Examples
+You can find a complete site with examples of Learnosity integration in our [demo site](http://demos.learnosity.com/).
+
+You can download the entire site or browse the code directly on [GitHub](https://github.com/Learnosity/learnosity-demos/).
+
+## Usage
+
+### Learnosity.init()
+
+The init function is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+
+The init function takes up to 5 arguments:
+
+ * [string]  service type
+ * [array]   security details (**no secret**)
```

Comment:
`[object]`, not `[array]`.

Reply:
Fixed here: https://github.com/Learnosity/learnosity-sdk-nodejs/pull/8/commits/06d0d5ed1f7c89fe8a0821359b97871fbad9cd4f